src/gazebo_mavlink_interface.cpp

```diff
@@ -412,8 +411,8 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
 
   if (_sdf->HasElement("imu_rate")) {
     imu_rate_ = _sdf->GetElement("imu_rate")->Get<int>();
+    imu_rate_ = 1/imu_rate_;
```

Comment:
@arwagoner where's the `set_imu_rate` variable which was set when the sdf element is detected?

Reply:
I deleted it because it seemed unnecessary.